components/Form/index.js

```diff
@@ -0,0 +1,103 @@
+import styled from "styled-components";
+
+import { useRouter } from "next/router";
+import useSWR from "swr";
+
+const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  width: 80%;
+  margin: auto;
+`;
+
+const StyledInput = styled.input`
+  border: 1px solid lightgrey;
+  border-radius: 15px;
+`;
+
+const StyledSelect = styled.select`
+  border: 1px solid lightgrey;
+  border-radius: 15px;
+`;
+
+export default function Form() {
+  const router = useRouter();
+  const { mutate } = useSWR("/api/projects");
+
+  async function handleClick(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const projectData = Object.fromEntries(formData);
+
+    const response = await fetch("/api/projects", {
+      method: "POST",
+      body: JSON.stringify(projectData),
+      headers: { "Content-Type": "application/json" },
+    });
+
+    if (response.ok) {
+      mutate();
+      router.push("/");
+    }
+  }
+  return (
+    <>
+      <StyledForm onSubmit={handleClick}>
+        <label for="title">Title: </label>
```

Comment:
I'm surprised ESLint didn't lose its marbles over `for` being used instead of `htmlFor`. I'm pretty sure this does nothing at the moment.
```suggestion
        <label htmlFor="title">Title: </label>
```